mandatory/1-fix-functions.js

```diff
@@ -33,20 +32,20 @@ function greaterThan10(num) {
 function sortArray(letters) {
   let sortedLetters = letters;
 
-  return sortedLetters;
+  return sortedLetters.sort();
 }
 
 function first5(numbers) {
-  let sliced;
+  let sliced = numbers;
 
-  return sliced;
+  return sliced.slice(0, 5);
 }
 
 function get3rdIndex(arr) {
   let index = 3;
-  let element;
+  let element = arr;
 
-  return element;
+  return element[index];
```

Comment:
same as above, move your change to line 46.

Reply:
@louisechow 

```
function get3rdIndex(arr) {
  let index = 3;
  let element = arr[index];

  return element;
}

```